templates/gitlab/mender-deployment-create.gitlab-ci.yml

```diff
@@ -0,0 +1,50 @@
+# Use this template to create deployments for Mender artifacts from MENDER_ARTIFACTS_FOLDER folder.
+#
+# The template is designed to be used with the includes keyword or copied into an existing
+# configuration. It requires `MENDER_SERVER_URL` and `MENDER_SERVER_ACCESS_TOKEN` variables
+# to work and can be used to create a deployment on a Mender server.
+#
+# It's possible to set custom values by setting the following environment variables:
```

Comment:
I wouldn't write "It's possible" because all of this are required. We require either MENDER_DEPLOYMENT_GROUP or MENDER_DEVICES_LIST. And we require MENDER_DEPLOYMENT_NAME and MENDER_RELEASE_NAME

Reply:
Yes. Good catch! This is outdated and came from the first implementation version, where default values where set by the template if the wars were not set by a user.